portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,38 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function showServerTime() {
+    const responseFromServer = await fetch('/date');
```

Comment:
[4.2](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation): Each time a new block or block-like construct is opened, the indent increases by two spaces. 

Here and elsewhere.

After you've set **Editor: Tab Size** to 2, you can use the **Format Document** action like you can with the HTML file.